compose/rest/record.go

```diff
@@ -567,6 +571,34 @@ func (ctrl *Record) TriggerScriptOnList(ctx context.Context, r *request.RecordTr
 	return api.OK(), err
 }
 
+func (ctrl *Record) Revisions(ctx context.Context, r *request.RecordRevisions) (interface{}, error) {
+	var (
+		makeRev = func() dal.ValueSetter { return &revisions.Revision{} }
+	)
+
+	iter, err := ctrl.record.SearchRevisions(ctx, r.NamespaceID, r.ModuleID, r.RecordID)
+	if err != nil {
+		return nil, err
+	}
+
+	return func(w http.ResponseWriter, req *http.Request) {
+		if _, err = w.Write([]byte(`{"set":[`)); err != nil {
+			return
+		}
+
+		err = dal.IteratorEncodeJSON(ctx, w, iter, makeRev)
+		if err != nil {
+			return
+		}
+
+		if _, err = w.Write([]byte(`]}`)); err != nil {
+			return
+		}
+
+		return
```

Comment:
I feel like this bit could be moved to some rest util. function (especially the opening and closing bytes

Reply:
A agree. As soon as this proves stable an we have a 2nd place were we can use it